impeller/playground/playground.cc

```diff
@@ -400,6 +400,54 @@ std::shared_ptr<Texture> CreateTextureForDecompressedImage(
     const std::shared_ptr<Context>& context,
     DecompressedImage& decompressed_image,
     bool enable_mipmapping) {
+  // TODO(https://github.com/flutter/flutter/issues/123468): copying buffers to
+  // textures is not implemented for GLES/Vulkan.
```

Comment:
The comment doesn't seem to align with the `#if`.  The if blocks a platform, the comment talks about backends.

Reply:
we don't have an ifdef for metal as far as I know. Technically macOS is only metal, except that we can run vulkan locally so this change breaks that without a modification to this code.

We can also fix this by finishing the blit pass implementation and removing the check altogether.

Reply:
Looks like this did break something, but hard to tell from CI logs.